ports/nrf/README.md

```diff
@@ -55,6 +55,14 @@ Alternatively the target board could be defined:
 
      make BOARD=pca10040
      make flash
+
+## Compile without LTO enabled
+
+As a space optimization, LTO (Link Time Optimization) has been enabled on all targets in the nrf-port. The `-flto` linker flag can be toggle easily by using the argument LTO when building. The example below shows how to disable LTO for the compilation:
```

Comment:
Would be good to wrap the words here at a line length of around 80 chars, to match other ports (and main README).  Also, "toggle" -> "toggled"